.pre-commit-config.yaml

```diff
@@ -47,7 +47,6 @@ repos:
     hooks:
       - id: remove-tabs
         types: [python]
-        exclude: ^Tools/c-analyzer/cpython/_parser.py
```

Comment:
@AA-Turner Shall we backport https://github.com/python/cpython/pull/137622 (that removed tabs from this file), or keep this exclude for 3.14 and older?

Reply:
Backport at https://github.com/python/cpython/pull/139718